show/bgp_quagga_v6.py

```diff
@@ -19,7 +19,11 @@ def bgp():
 @bgp.command()
 def summary():
     """Show summarized information of IPv6 BGP state"""
-    run_command('sudo vtysh -c "show ipv6 bgp summary"')
+    try:
+        device_output = run_command('sudo vtysh -c "show ipv6 bgp summary"', return_cmd=True)
+        get_bgp_summary_extended(device_output)
+    except:
```

Comment:
What exception could we receive here?

Reply:
I added this block as a failsafe. If anything goes wrong in processing the output, atleast we will get the older output.
Shall we keep it this way or change ?